CHANGELOG.md

```diff
@@ -82,6 +84,8 @@
 - Segregated MMDS documentation in MMDS design documentation and MMDS user
   guide documentation.
 - The logger `level` field is now case-insensitive.
+- MicroVM process is only attached to the cgroups defined by using `--cgroups`
```

Comment:
Hello @iulianbarbu, maybe I'm loosing something, but as far as I can tell, this is added under Changed not under Fixed. I added this to Changed because of this paragraph at the jailer docs:

```
The jailer then writes the current pid to /sys/fs/cgroup/cpu/firecracker/551e7604-e35c-42b3-b825416853441234/tasks,
/sys/fs/cgroup/cpuset/firecracker/551e7604-e35c-42b3-b825-416853441234/tasks, and
/sys/fs/cgroup/pids/firecracker/551e7604-e35c-42b3-b825-416853441234/tasks. It also writes 0 to
/sys/fs/cgroup/cpuset/firecracker/551e7604-e35c-42b3-b825-416853441234/cpuset.mems.
```

With my changes, pid is only written in the tasks file of the cgroups which are set by using `--cgroups` or `--node`.